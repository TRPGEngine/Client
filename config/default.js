/**
 * NOTE: 对于app来说。需要重新编译二进制版本后JS文件才能取到配置
 * 因此，增加新的配置（如果该配置在JS中用到）需要发布二进制文件
 */

const appName = process.env.TRPG_APP_NAME || 'Game';
const environment = process.env.NODE_ENV;
const package = require('../package.json');

module.exports = {
  release: {
    // 生产环境必填
    store: {
      file: '',
      password: '',
    },
    key: {
      alias: '',
      password: '',
    },
  },
  codepush: {
    url: 'http://codepush.moonrailgun.com',
    deploymentKey: 'pafmXkZi2xoKkj7UkNrGijKJJu2c4ksvOXqog', // 正式环境
    deploymentKeyStaging: 'puSxJ4RphFExKxLQQLvpI27e8C8r4ksvOXqog', // 生产环境
  },
  sentry: {
    dsn: '',
    mobileDsn: '', // 移动端的dsn
    feedbackUrl: '', // 见https://docs.sentry.io/api/projects/post-project-user-reports/
    pushRelease: false, // 仅在production环境使用该变量
    release: `${appName}-v${package.version}-${environment}`,
  },
  posthog: {
    token: '',
    instance: '',
  },
  umeng: {
    push: {
      channel: 'TRPG',
      appkey: '',
      messageSecret: '',
    },
    mipush: {
      // 小米厂商推送通道配置
      appid: '',
      appkey: '',
    },
  },
};
