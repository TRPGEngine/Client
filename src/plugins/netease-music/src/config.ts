/**
 * 网易云音乐接口地址
 */
export const neteaseMusicAPI = 'https://musicapi.moonrailgun.com';
