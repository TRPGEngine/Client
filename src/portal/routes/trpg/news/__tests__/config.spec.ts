import config from '@shared/project.config';

test('newsRSSUrl', () => {
  expect(config.url.rssNews).toMatchSnapshot();
});
