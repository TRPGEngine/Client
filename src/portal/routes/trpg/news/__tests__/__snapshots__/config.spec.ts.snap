// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`newsRSSUrl 1`] = `
Array [
  Object {
    "name": "贴吧",
    "url": "https://rss.moonrailgun.com/tieba/forum/跑团?code=42bacec52565b92308da5f98ec1509c9",
  },
]
`;
